apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/RetryHandler.cs

```diff
@@ -28,6 +28,7 @@ namespace Google.Cloud.Storage.V1
     /// </summary>
     internal sealed class RetryHandler : IHttpUnsuccessfulResponseHandler
     {
+        // Retriable error codes: 408—Request Timeout, 429-Too Many Requests, 500-Internal Server Error, 502—Bad Gateway, 503—Service Unavailable, 504—Gateway Timeout
```

Comment:
I'd probably put these into multiple lines:

```csharp
// Retriable error codes:
// - 408: Request Timeout
// - 429: Too Many Requests
// - 500: Internal Server Error
// - 502: Bad Gateway
// - 503: Service Unavailable
// - 504: Gateway Timeout
```

An alternative formatting would be to comment inline in the array declaration:

```csharp
private static readonly int[] s_retriableErrorCodes = { 408, 429, 500, 502, 503, 504 };
{
    408, // Request timeout
    429, // Too many requests
    500, // Internal server error
    502, // Bad gateway
    503, // Service unavailable
    504, // Gateway timeout
};
```

I *think* that's probably better, as it makes it harder for the comment and array content to get out of sync... but see what you think.